pkg/distributor/distributor.go

```diff
@@ -517,7 +532,7 @@ func (d *Distributor) queryIngesters(ctx context.Context, ingesters []*ring.Inge
 					}
 					fpToSampleStream[fp] = mss
 				}
-				mss.Values = util.MergeSamples(mss.Values, ss.Values)
+				mss.Values = util.MergeSampleSets(mss.Values, ss.Values)
```

Comment:
We changed it to `MergeSampleSets` to keep it consistent with `MergeNSampleSets` and it's only used in 2 places. I believe this was in a previous PR but we have now agreed to merge all PR's together.